.scalafmt.conf

```diff
@@ -0,0 +1,9 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more contributor
+# license agreements; and to You under the Apache License, Version 2.0.
+
+style = intellij
+danglingParentheses = false
+maxColumn = 120
+docstrings = JavaDoc
+rewrite.rules = [SortImports]
+project.git = true
```

Comment:
Looks like the first step to enabling scalafmt in this repo

I can add the actual check to Travis once this is merged

Reply:
Thanks @drcariel 👍